poetry.lock

```diff
@@ -1654,71 +1654,74 @@ test = ["enum34", "ipaddress", "mock", "pywin32", "wmi"]
 
 [[package]]
 name = "psycopg2-binary"
-version = "2.9.3"
```

Comment:
Let's also bump the version in https://github.com/neondatabase/neon/blob/ff1a1aea86b0245ad039039ed4c86591270ca567/pyproject.toml#L10

A note, in tests, we still use Python 3.9, which works fine with 2.9.3 (even would work with 2.9.1, I think); you probably use Python 3.11 and such old `psycopg2-binary` doesn't provide wheels for this version (python 3.11 isn't compatible with some of the tools that we use in [Cloud repo](https://github.com/neondatabase/cloud/pull/5022)), jfyi.